CHANGES.rst

```diff
@@ -122,6 +123,8 @@ better with 2d coadds.
 1.10.0 (11 July 2022)
 ---------------------
 
+1.9.2dev
+--------
```

Comment:
Something weird happened in the merge here.

Reply:
greed, removed